SOURCES.toml

```diff
@@ -0,0 +1,686 @@
+[[resources]] # Path: 7zip.png
+path        = "7zip.png"
+description = """Application icons from open-source software projects."""
+
+[[resources]] # Path: airports.csv
+path = "airports.csv"
+
+[[resources]] # Path: annual-precip.json
+path        = "annual-precip.json"
+description = """A raster grid of global annual precipitation for the year 2016 at a resolution 1 degree of lon/lat per cell."""
+
+[[resources.sources]]
+title = "Climate Forecast System Version 2"
+path  = "https://www.ncdc.noaa.gov/data-access/model-data/model-datasets/climate-forecast-system-version2-cfsv2"
+
+[[resources]] # Path: anscombe.json
+path        = "anscombe.json"
+description = """Graphs in Statistical Analysis, F. J. Anscombe, The American Statistician."""
+
+[[resources]] # Path: barley.json
+path        = "barley.json"
+description = """The result of a 1930s agricultural experiment in Minnesota, this dataset contains yields for 10 different varieties of barley at six different sites. It was first published by agronomists F.R. Immer, H.K. Hayes, and L. Powers in the 1934 paper \"Statistical Determination of Barley Varietal Adaption.\" R.A. Fisher's popularized its use in the field of statistics when he included it in his book \"The Design of Experiments.\" Since then it has been used to demonstrate new statistical techniques, including the trellis charts developed by Richard Becker, William Cleveland and others in the 1990s."""
```

Comment:
```suggestion
[[resources]] # Path: barley.json
path = "barley.json"
description = """The result of a 1930s agricultural experiment in Minnesota, this dataset contains yields for 10 different varieties of barley at six different sites.
It was first published by agronomists F.R. Immer, H.K. Hayes, and L. Powers in the 1934 paper \"Statistical Determination of Barley Varietal Adaption\".
R.A. Fisher's popularized its use in the field of statistics when he included it in his book \"The Design of Experiments\".
Since then it has been used to demonstrate new statistical techniques, including the trellis charts developed by Richard Becker, William Cleveland and others in the 1990s.
"""
```